Automattic/jetpack
# WAF: Add brute force protection tests

#### Description
Moves Protect module tests out of the Jetpack plugin and into the WAF package.

## Proposed changes:

* Adds Brute force shared functions tests to the WAF package
* Adjusts the original tests and adds functions to account for those that are now undefined 
* Removes the Protect module tests from the Jetpack plugin

### Other information:


## Jetpack product discussion


- 1201069996155224-as-1203870604385748

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout the branch
* Open a terminal window and run `cd projects/packages/waf`
* Run all PHPUnit tests via `composer phpunit`
* Verify that all tests pass
